app/assets/javascripts/code/controllers/_sessions_list_ctrl.js

```diff
@@ -102,14 +102,6 @@ export const SessionsListCtrl = (
     return {id: params.get('tmp').selectedSensorId, heat: params.get('data').heat };
   };
 
-  $scope.$watch("sessionRedrawCondition()", function(newValue) {
-    console.log("watch - sessionRedrawCondition()");
-    if(singleSession.isFixed() || (!newValue.id && !newValue.heat)){
-      return;
-    }
-    drawSession.redraw(sessions.allSelected());
-  }, true);
-
```

Comment:
I'm not sure if the condition on lines 101-103 is necessary for anything else, or should I delete it as well?

Reply:
If you can't find it with a grep go ahead and remove ;)

Reply:
`params.get('tmp').selectedSensorId` could be connected to selecting sessions when there is all / all and the user uses the dialog. You may want to test that case.

It's interesting we don't need this anymore

Reply:
Yes. I was just as surprised, but it works the same without this watch...